SUMMARY.md

```diff
@@ -24,6 +24,14 @@
     * [Forking Ocean Market](building-with-ocean/build-a-marketplace/forking-ocean-market.md)
     * [Customising your market](building-with-ocean/build-a-marketplace/customising-your-market.md)
     * [Deploying your market](building-with-ocean/build-a-marketplace/deploying-market.md)
+  * [Obtaining Ethereum node provider API key](building-with-ocean/obtaining-api-key.md)
+  * [Using Ocean libraries](building-with-ocean/using-ocean-libraries/README.md)
+    * [Configuration](building-with-ocean/using-ocean-libraries/configuration.md)
+    * [Creating a data NFT](building-with-ocean/using-ocean-libraries/creating\_dataNFT.md)
+    * [Create datatoken with fixed pricing](building-with-ocean/using-ocean-libraries/create-datatoken-with-fixed-pricing.md)
+    * [Mint datatoken](building-with-ocean/using-ocean-libraries/mint-datatoken.md)
+    * [Update metadata](building-with-ocean/using-ocean-libraries/update-metadata.md)
+  * [Publish assets using hosting services](building-with-ocean/asset-hosting.md)
```

Comment:
This link is now in here twice, was that deliberate? It's also in the "how to use ocean market" section, I would suggest we just keep it in that section

Reply:
Thanks for catching this! The link was deliberately left while resolving conflict.